src/common/fibonacci.ts

```diff
@@ -0,0 +1,9 @@
+export const isPerfectSquare = (n: number) => {
+  return Number.isInteger(Math.sqrt(n));
+};
+
+// Number is Fibonacci if (5*n^2 + 4) or (5*n^2 - 4) is a perfect square
```

Comment:
Nice!! Didn't know that

Reply:
TIL

Reply:
I also learned it when googling how to test if a number belongs to the fibonacci sequence :)